elodie/filesystem.py

```diff
@@ -10,6 +10,7 @@
 import re
 import shutil
 import time
+import locale
```

Comment:
Can these changes go into geolocation.py instead of filesystem.py?

Reply:
but its in filesystem.py `strftime` is used?

Reply:
locale is imported to translate `strftime` *(mostly month names)* and that is happening in filesystem.py